staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -104,6 +104,7 @@ const (
 const (
 	FinalizerOrphanDependents = "orphan"
 	FinalizerDeleteDependents = "foregroundDeletion"
+	FinalizerKubernetes       = "kubernetes"
```

Comment:
this isn't defined for metav1 like the garbage collection finalizers are, let's drop this line

Reply:
still outstanding, please drop this line